lib/value.js

```diff
@@ -27,13 +27,7 @@ class Value extends EventEmitter {
   constructor(initialValue, valueForwarder) {
     super();
     this.lastValue = initialValue;
-    if (!valueForwarder) {
```

Comment:
On line 24 above it should be documented that the valueForwarder method is optional

Reply:
Fixed.